jstests/libs/override_methods/inject_tenant_prefix.js

```diff
@@ -172,6 +172,12 @@ function removeTenantIdFromString(string) {
     return string.replace(new RegExp(`${TestData.tenantId}_`, 'g'), "");
 }
 
+function isShardSplitPassthrough(conn) {
+    const doc =
+        originalRunCommand.apply(conn, ["admin", {getParameter: 1, shardSplitGarbageCollectionDelayMS: 1}, 0]);
+    return doc.shardSplitGarbageCollectionDelayMS && doc.shardSplitGarbageCollectionDelayMS === 1;
```

Comment:
Open to alternatives on this one. I noticed that all shard split passthroughs set this to `1`, so it felt like an easy way to avoid changing all of those files to put an explicit `shardSplitPassthrough: true` in `TestData`